client/src/components/AboutUkraine/AboutUkraine.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import {Box, Typography, styled, keyframes  } from '@mui/material';
+// import { useInView } from 'react-intersection-observer';
+
+const UkraineTextInfo = styled((props) => <Typography variant="body1" {...props} />)(({ theme }) => ({
+    [theme.breakpoints.up('tablet')]: {
+        padding: '10px',
+        fontSize: '18px',
+        lineHeight: '1.2',
+    },
+
+    [theme.breakpoints.up('laptop')]: {
+        fontSize: '18px',
```

Comment:
the same as the above. delete the fontsize property, you can leave the padding.

please check our theme.js file for reference.

Reply:
done